streamwebs_frontend/streamwebs/templates/streamwebs/sites.html

```diff
@@ -62,5 +62,9 @@
             {% endfor %}
             </ul>
         </div>
+        <br>
+        <div id="create-site">
+            <a href="{% url 'streamwebs:create_site' %}" class="btn teal darken-4">CREATE_SITE</a>
```

Comment:
Nah, that's fine as is. Just change the text for the button since it's currently the only one that uses an underscore instead of a space. We should keep things consistent.

![buttons](https://user-images.githubusercontent.com/6801364/26897990-449c9ada-4b7f-11e7-97f2-7ae8a97dcc16.png)